server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -230,13 +226,7 @@ public static void validateIndexOrAliasName(String index, BiFunction<String, Str
         if (index.charAt(0) == '_' || index.charAt(0) == '-' || index.charAt(0) == '+') {
             throw exceptionCtor.apply(index, "must not start with '_', '-', or '+'");
         }
-        int byteCount = 0;
-        try {
-            byteCount = index.getBytes("UTF-8").length;
-        } catch (UnsupportedEncodingException e) {
-            // UTF-8 should always be supported, but rethrow this if it is not for some reason
-            throw new ElasticsearchException("Unable to determine length of index name", e);
```

Comment:
Just checking, Is this re-throwing just bogus here? And therefor can be removed?

Reply:
This was a checked exception actually for when you pass an encoding name that doesn't exist. So yea, this should've probably been an assertion before as well and was bogus. If you use the `StandardCharset` it can't throw though and all of this goes away :)

Reply:
>  If you use the StandardCharset it can't throw though and all of this goes away

I had to look again to see the use `StandardCharset`.

👍